src/main/java/io/wispforest/lavender/structure/StructureTemplate.java

```diff
@@ -49,8 +75,8 @@ public StructureTemplate(Identifier id, BlockStatePredicate[][][] predicates, in
         this.zSize = zSize;
 
         this.anchor = anchor != null
-            ? anchor
-            : new Vec3i(this.xSize / 2, 0, this.ySize / 2);
+                ? anchor
```

Comment:
Why did this indent change?

Reply:
ah didn't realize the indent change happened. that was my autoformatter settings.

do you think you could commit a `.editorconfig` file to the repo, so that my formatter settings can mirror yours?

intellij's "editorconfig" plugin allows you to autogenerate one, and when opening a project with a `.editorconfig` it applies those settings.